test/Execute.py

```diff
@@ -86,7 +86,7 @@
 if sys.platform == 'win32' and sys.version_info[0] == 2:
     expect = r"""scons: \*\*\* Error 1
 scons: \*\*\* Error 2
-scons: \*\*\* nonexistent.in/\*\.\*: (The system cannot find the path specified|Das System kann den angegebenen Pfad nicht finden)"""
+scons: \*\*\* nonexistent.in\\\*\.\*: (The system cannot find the path specified|Das System kann den angegebenen Pfad nicht finden)"""
```

Comment:
I can confirm this change does fix the Execute.py fail  for Python 2.7. Be nice to get it in, even if the rest of the patch is held up. I'm still not clear on why scons provides different error messages in this case (python2 vs. python3) - is that error message from python itself?

Reply:
I might separate the patch to remove the stuff I have not had time to make a good test case for as of yet. Logically I know it is a good fix. I need to reproduce the case again in which I had this crash.